src/protocols/bitcoin/views/AccountDetailsTransactions.vue

```diff
@@ -101,7 +96,7 @@ export default defineComponent({
 
     return {
       isOnline,
-      transactions,
+      transactions: loadedTransactionList,
```

Comment:
By following the naming pattern for two next lines: `transactionsLatest` and `transactionsMore` the name used in this line should be: `transactionsLoaded`.